src/test/java/underdogs/devbie/notice/acceptance/NoticeAcceptanceTest.java

```diff
@@ -53,8 +55,28 @@
          Then 공고가 삭제되었다.
      */
     @DisplayName("공고 인수테스트")
-    @Test
-    void notice() throws JsonProcessingException {
+    @TestFactory
+    Stream<DynamicTest> notice() {
+        return Stream.of(
+            DynamicTest.dynamicTest("공고 게시글을 생성한다.", () -> {
+                createNotice();
+            }),
+            DynamicTest.dynamicTest("공고 게시글 전체를 조회한다.", () -> {
+                readAllNotice();
+            }),
+            DynamicTest.dynamicTest("공고 게시글을 수정한다.", () -> {
+                updateNotice();
+            }),
+            DynamicTest.dynamicTest("공고 게시글 하나르 상세 조회한다.", () -> {
+                readNoticeDetail();
+            }),
+            DynamicTest.dynamicTest("공고 게시글을 삭제한다.", () -> {
+                deleteNotice();
+            })
```

Comment:
👍👍
```suggestion
            dynamicTest("공고 게시글을 생성한다.", () -> {
                createNotice();
            }),
            dynamicTest("공고 게시글 전체를 조회한다.", () -> {
                readAllNotice();
            }),
            dynamicTest("공고 게시글을 수정한다.", () -> {
                updateNotice();
            }),
            dynamicTest("공고 게시글 하나르 상세 조회한다.", () -> {
                readNoticeDetail();
            }),
            dynamicTest("공고 게시글을 삭제한다.", () -> {
                deleteNotice();
            })
```

static import를 하면 보기 좋은 코드가 될 것 같아요!

Reply:
네 반영했습니다!